exchange/exchangetest/firstpartydata-imp-ext-multiple-prebid-bidders.json 

```diff
@@ -61,7 +61,6 @@
                             "bidder": {
                                 "placementId": 1
                             },
-                            "prebid": {},
```

Comment:
Previously, `splitImps` would keep an empty prebid json section. The refactor removes it entirely now. I thought that would be a fine change. Do you agree? .. or should I keep the older functionality?